server/main.go

```diff
@@ -64,14 +72,14 @@ func NewApp(config *datastore.ServerConfig) App {
 }
 
 // Read the config.yml file
-func ReadConfig(file string) (*datastore.ServerConfig, error) {
```

Comment:
You want to still read `ServerConfig` actually. But now bring `ServerConfig` back to `main.go` and modify it so that it maps to the new `config.yml` format. Think @mikelu92  might have to walk you through it if you don't get how to do it.